packages/mux-player/src/template.ts

```diff
@@ -72,6 +72,7 @@ export const template = (props: MuxTemplateProps) => html`
           />`
         : ""}
     </mux-video>
+    <div slot="gestures-chrome" class="mxp-background"></div>
```

Comment:
Putting it in the `gestures-chrome` allows it to be in front of the media player but before the controls.

Reply:
went with a css-only solution.